### PR TITLE
research task

### PR DESCRIPTION
**Pull request by Rue-ops6**

### CODE REVIEW

File: Research content/middleware.md
Comment (Nora-Lootfy) [inline Research content/middleware.md:1]: images

File: Research content/otherWEBdirectory.md
Comment (Nora-Lootfy) [inline Research content/otherWEBdirectory.md:1]: this is an old method, was used until laravel 10, when you search make sure you write laravel 11, but don't worry actually you almost reach the correct solution, good job @Rue-ops6

**Review by Nora-Lootfy — APPROVED** [review]
Thank ya @Rue-ops6

File: Research content/middleware.md
Comment (Rue-ops6) [reply Research content/middleware.md:1]: يلهوي اخيرا عملت حاجه صح 💃🏼💃🏼💃🏼

File: Research content/otherWEBdirectory.md
Comment (Rue-ops6) [reply Research content/otherWEBdirectory.md:1]: تصدقي حاجه زي دي مجتش فبالي 
تخيلت هيجبلب اجدد حاجه
مقرتش صراحة فشكرا:""

**Pull request merged**